js/gc-smart-data.js

```diff
@@ -406,7 +406,10 @@ window.GC = window.GC || {};
         if ( d.valid() ) {
             this.DOB = d;
             this.birthdate = d.toString();
-            this.gestationAge = this.weeker = Math.round(40 - this.DOB.diffWeeks(this.EDD));
+
+            if (this.gestationAge == null) {
```

Comment:
Cerner gives us the pre-calculated gest age in their fhir service response. 
There is sometimes a discrepancy of 1 day when using the calculation, so this logic is guarding against changing the gest age, if we have already read and populated it elsewhere. 
is the DOB or EDD likely to change? and if so, wont restarting the app simply realign all the values?